### PR TITLE
Send create and update requests to miq controller

### PR DESCRIPTION
**Pull request by brittballard**
I know there is discussion about whether we want to break this out into two controllers, but I suggest we merge this fix for the time being. There are lots of questions that have to be answered before it's even possible to start editing products other than MIQ products via the client.

### CODE REVIEW

**Comment by chriskacerguis** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/54d4fa726956803255002db9/compare/bb-miq-controller">analyzed this pull request</a>.

**Comment by brittballard** [comment]
Requires https://github.com/projectjellyfish/api/pull/338

File: assets/js/app/products/products_resource.js [hunk 1]
```diff
@@ -4,7 +4,12 @@
 var ProductsResource = function($resource, apiResource) {
   return $resource(apiResource('productsById'), {'id': '@id'}, {
     'update': {
-      method: 'PUT'
+      method: 'PUT',
```
Comment (stackus) [inline assets/js/app/products/products_resource.js]: Since is a temporary fix could you include a TODO with an explanation/link to discussion.

Comment (brittballard) [inline assets/js/app/products/products_resource.js]: Sure.

**Pull request merged**